spec/forms/password_reset_email_form_spec.rb

```diff
@@ -12,29 +12,33 @@
         user = build(:user, :signed_up, email: 'test1@test.com')
         subject = PasswordResetEmailForm.new('Test1@test.com')
 
-        result = {
-          success: true,
-          errors: [],
+        extra = {
           user_id: user.uuid,
           role: user.role,
           confirmed: true,
         }
 
+        result = instance_double(FormResponse)
+
+        expect(FormResponse).to receive(:new).
```

Comment:
We want to verify that the result is an instance of FormResponse, and that the instance received the correct arguments. What was the alternative you were thinking about?